modular_skyrat/master_files/code/_globalvars/maint_loot_uncommon.dm

```diff
@@ -319,5 +319,9 @@ GLOBAL_LIST_INIT(uncommon_loot, list(//uncommon: useful items
 		/obj/item/eldritch_potion/crucible_soul = 1,
 		/obj/item/eldritch_potion/duskndawn = 1,
 		/obj/item/eldritch_potion/wounded = 1
-	) = 1
+	) = 1,
+	list( //Modsuits
+		/obj/item/mod/construction/broken_core = 1,
+		/obj/effect/spawner/random/mod/maint = 3
```

Comment:
```suggestion
		/obj/effect/spawner/random/mod/maint = 3,
```

Reply:
the code does not like it if you put a comma on the last item in a list